src/names/preludes.md

```diff
@@ -18,9 +18,22 @@ There are several different preludes:
 
 ## Standard library prelude
 
-The standard library prelude includes names from the [`std::prelude::v1`]
-module. If the [`no_std` attribute] is used, then it instead uses the names
-from the [`core::prelude::v1`] module.
+Each crate has a standard library prelude, which consists of the names from a single standard library module. The module used depends on the crate's edition, and on whether the [`no_std` attribute] is applied to the crate:
+
+Edition | `no_std` not applied        | `no_std` applied
+--------| --------------------------- | ----------------------------
+2015    | [`std::prelude::rust_2015`] | [`core::prelude::rust_2015`]
+2018    | [`std::prelude::rust_2018`] | [`core::prelude::rust_2018`]
+2021    | [`std::prelude::rust_2021`] | [`core::prelude::rust_2021`]
+
+This choice of prelude is in effect throughout the crate, including code generated by macros defined in crates with a different edition or `no_std` attribute.
```

Comment:
I'm having a bit of trouble fully understanding what this sentence is trying to say.  Might I suggest to just remove it? The start of this chapter describes how every prelude is part of name resolution for every module.  Also, there isn't anything special about the standard library prelude with respect to macros.  

In the future, once [Name Resolution](https://doc.rust-lang.org/nightly/reference/names/name-resolution.html) is filled out, it might become more clear how names from preludes are resolved. Also, I think the [Hygiene section](https://doc.rust-lang.org/nightly/reference/macros-by-example.html#hygiene) explains how names are resolved in macros.

Reply:
I've removed it